zcash_client_sqlite/Cargo.toml

```diff
@@ -13,19 +13,24 @@ license = "MIT OR Apache-2.0"
 edition = "2018"
 
 [dependencies]
+async-trait = "0.1.52"
 bech32 = "0.9.1"
 bs58 = { version = "0.4", features = ["check"] }
 ff = "0.8"
+futures = { version = "0.3" }
 group = "0.8"
 jubjub = "0.5.1"
 protobuf = "2.20"
 rand_core = "0.5.1"
 rusqlite = { version = "0.28", features = ["time"] }
 libsqlite3-sys= { version = "0.25.2", features = ["bundled"] }
-time = "0.3"
+time = { version = "0.3.20", features = ["wasm-bindgen"]}
```

Comment:
is `wasm-bindgen` needed for WASM? If so, you can enable that target when compiling for WASM with `[target.'cfg(target_arch = "wasm32")'.dependencies]`